main.nf

```diff
@@ -249,13 +289,24 @@ process rnaseqc {
     read_duplication.py -i $bam4 -o ${bam4}.read_duplication
     RPKM_saturation.py -i $bam4 -r $bed12 -d '1+-,1-+,2++,2--' -o ${bam4}.RPKM_saturation
     """
+    
+    }else{
+ 
+    """
+    bam_stat.py -i $bam4 2> ${bam4}.bam_stat.txt
+    junction_annotation.py -i $bam4 -o ${bam4}.rseqc -r $bed12
+    junction_saturation.py -i $bam4 -o ${bam4}.rseqc -r $bed12
+    inner_distance.py -i $bam4 -o ${bam4}.rseqc -r $bed12
+    geneBody_coverage.py -i $bam4 -o ${bam4}.rseqc -r $bed12
+    infer_experiment.py -i $bam4 -r $bed12 > ${bam4}.infer_experiment.txt
+    read_distribution.py -i $bam4 -r $bed12 > ${bam4}.read_distribution.txt
+    read_duplication.py -i $bam4 -o ${bam4}.read_duplication
+    RPKM_saturation.py -i $bam4 -r $bed12 -d '++,--' -o ${bam4}.RPKM_saturation
```

Comment:
Is this line the only one that changes? If so it would be cleaner to put the `'++,--'` into a variable using conditionals above and then only have one set of commands..